leapp/libraries/stdlib/call.py

```diff
@@ -94,25 +97,35 @@ def _call(command, callback_raw=lambda fd, value: None, callback_linebuffered=la
                                  The default value of 80 chosen to correspond with suggested terminal line width
         :type read_buffer_size: int
         :param callback_raw: Callback executed on raw data (before decoding) as they are read from file descriptors
-        :type callback_raw: (fd: int, buffer: bytes) -> None
+        :type callback_raw: ((fd: int, fd_type: int), buffer: bytes) -> None
         :param callback_linebuffered: Callback executed on decoded lines as they are read from the file descriptors
-        :type callback_linebuffered: (fd, buffer) -> None
+        :type callback_linebuffered: ((fd: int, fd_type: int), value: str) -> None
         :param stdin: String or a file descriptor that will be written to stdin of the child process
         :type stdin: int, str
+        :param env: Environment variables to use for execution of the command
+        :type env: dict
         :return: {'stdout' : stdout, 'stderr': stderr, 'signal': signal, 'exit_code': exit_code, 'pid': pid}
         :rtype: dict
     """
     if not isinstance(command, (list, tuple)):
         raise TypeError('command parameter has to be a list or tuple')
-    if not callable(callback_raw) or callback_raw.__code__.co_argcount != 2:
-        raise TypeError('callback_raw parameter has to be callable accepting 2 parameters')
-    if not callable(callback_linebuffered) or callback_linebuffered.__code__.co_argcount != 2:
+    if not callable(callback_raw) or\
+            (getattr(callback_raw, '__code__', None) and callback_raw.__code__.co_argcount != 2):
+            raise TypeError('callback_raw parameter has to be callable accepting 2 parameters')
+    if not callable(callback_linebuffered)\
+            or (getattr(callback_linebuffered, '__code__', None)
+                and callback_linebuffered.__code__.co_argcount != 2):
```

Comment:
```suggestion
                and callback_linebuffered.__code__.co_argcount != 2): # noqa
```

should ignore flake8 error